src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx

```diff
@@ -326,11 +326,11 @@ export default class SecurityUserSettingsTab extends React.Component<IProps, ISt
         return (
             <div className="mx_SettingsTab mx_SecurityUserSettingsTab">
                 { warning }
-                <div className="mx_SettingsTab_heading">{ _t("Where you’re logged in") }</div>
+                <div className="mx_SettingsTab_heading">{ _t("Where you’re signed in") }</div>
```

Comment:
we try to avoid magic/stylistic quotes - not sure if there are more in other places, but this one caught my eye.

Reply:
Sorry I missed this comment before merging; I'm opening up a separate PR with a bulk find-replace of all the magic quotes in the codebase.